config.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
+
+module.exports = {
+  PRETTY_DATE_FORMAT: 'DD MMMM YYYY',
+  dateTimeFormat: 'DD MMM YYYY HH:mm:ss',
+  env: env,
+  govukNotify: {
+    notifyApiKey: process.env.NOTIFY_KEY,
+    caseworkerEmail: process.env.FEEDBACK_INBOX,
+    userConfirmationTemplateId: process.env.FEEDBACK_TEMPLATE_ID
+  },
+  hosts: {
+    acceptanceTests: process.env.ACCEPTANCE_HOST_NAME || `http://localhost:${process.env.PORT || 8080}`
+  },
+  redis: {
+    port: process.env.REDIS_PORT || '6379',
+    host: process.env.REDIS_HOST || '127.0.0.1'
+  },
+  sessionDefaults: {
+    steps: ['/feedback', '/feedback-sent']
+  }
```

Comment:
It's not clear why all the steps should be defaults?

Reply:
Going to remove this prop. Think it's mainly used for passing step data from app to app